src/divideBy.js

```diff
@@ -0,0 +1,20 @@
+import { divide, flip } from 'ramda';
+
+/**
+ * Divides two numbers, where the second number is divided by the first number.
+ *
+ * @func divideBy
```

Comment:
I'm struggling with the name of the function. In original ticket I've suggested to use `divideNum`. The thing is that `By` and `With` suffixes on functions are reserved for special behavior described in [this issue](https://github.com/char0n/ramda-adjunct/issues/653). By looking at the name of the function I'd expect that it consumes unary function as a first argument. I'd prefer `divideNum` even though it's "uglier" then `divideBy`.

Reply:
That makes sense... Thanks for the explanation.